property.go

```diff
@@ -0,0 +1,91 @@
+package opslevel
+
+import "fmt"
+
+type PropertyDefinitionInput struct {
+	Name   string     `json:"name"`
+	Schema JSONString `json:"schema"`
+}
+
+type PropertyDefinition struct {
+	Aliases []string `graphql:"aliases"`
+	Id      ID       `graphql:"id"`
+	Name    string   `graphql:"name"`
+	Schema  JSON     `graphql:"schema" scalar:"true"`
+}
+
+type PropertyDefinitionConnection struct {
+	Nodes      []PropertyDefinition
+	PageInfo   PageInfo
+	TotalCount int `graphql:"-"`
+}
+
+func (client *Client) CreatePropertyDefinition(input PropertyDefinitionInput) (*PropertyDefinition, error) {
+	var m struct {
+		Payload struct {
+			Definition PropertyDefinition `graphql:"definition"`
+			Errors     []OpsLevelErrors   `graphql:"errors"`
+		} `graphql:"propertyDefinitionCreate(input: $input)"`
+	}
+	v := PayloadVariables{
+		"input": input,
+	}
+	err := client.Mutate(&m, v, WithName("PropertyDefinitionCreate"))
+	return &m.Payload.Definition, HandleErrors(err, m.Payload.Errors)
+}
+
+func (client *Client) GetPropertyDefinition(input string) (*PropertyDefinition, error) {
+	var q struct {
+		Account struct {
+			Definition PropertyDefinition `graphql:"propertyDefinition(input: $input)"`
+		}
+	}
+	v := PayloadVariables{
+		"input": *NewIdentifier(input),
+	}
+	err := client.Query(&q, v, WithName("PropertyDefinitionGet"))
+	if q.Account.Definition.Id == "" {
+		err = fmt.Errorf("PropertyDefinition with ID or Alias matching '%s' not found", input)
```

Comment:
Let's remove `or Alias` from the error message since it's only checking for the `Id`

Reply:
no the input to the function can take an alias or id - thats what its saying.  This is all good.